sfc/application.txt

```diff
@@ -0,0 +1,632 @@
+
+    - Emeric Fermas <emeric.fermas@gmail.com>, active 2011 - present
+      Employed by AXA (2011 - present)
+      Contracted by GitHub Inc (2012 - present)
+
+    - Philip Kelley <phkelley@hotmail.com>, active 2012 - 2014
+      Employed by Microsoft Corp (2012 - 2014)
+
+    - Jacques Germishuys <jacquesg@striata.com>, active 2014 - 2015
+      Employed by Striata Marketing Pty (2014 - 2015)
+
+    - Vicent Martí <vicent@github.com>, active 2010 - present
+      Google Summer of Code Student (2010)
+      Contracted by GitHub Inc (2010 - 2011)
+      Employed by GitHub Inc (2011 - present)
+
+    - Carlos Martín Nieto <cmn@dwim.me>, active 2011 - present
+      Google Summer of Code Student (2011)
+      Employed by elego Software Solutions GmbH (2011 - 2015)
+      Contracted by elego, GitHub Inc, Microsoft Corp (2015 - 2016)
+      Employed by GitHub Inc (2016 - present)
+
+    - Shawn Pearce <spearce@spearce.org>, active 2008 - 2009
+      Employed by Google (2008 - 2009)
+
+    - Arthur Schreiber <schreiber.arthur@gmail.com>, active 2012 - present
+      Employed by Hewlett-Packard (2012 - 2016)
+      Employed by GitHub (2016 - present)
+
+    - Michael Schubert <schu@schu.io>, active 2011 - 2013
+      Google Summer of Code Student (2012)
+      Employed by elego Software Solutions GmbH (2011 - 2013)
+
+    - Patrick Steinhardt <ps@pks.im>, active 2015 - present
+      Employed by elego Software Solutions GmbH (2015 - present)
+
+    - Ben Straub <bs@github.com>, active 2012 - 2014
+      Employed by GitHub Inc (2012 - 2014)
+
+    - Edward Thomson <ethomson@edwardthomson.com>, active 2012 - present
+      Employed by Microsoft Corp (2012 - 2015)
+      Employed by GitHub Inc (2016 - 2017)
+      Employed by Microsoft Corp (2017 - present)
+
+    Contributors to the LibGit2Sharp project (who are not included
+    above):
+
+    - Tim Clem <timothy.clem@gmail.com>, active 2011 - 2012
+      Employed by GitHub Inc (2011 - 2012)
+
+    - Keith Dahlby <dahlbyk@gmail.com>, active 2011 - present
+      Employed by J&P Cycles (2011 - 2016)
+      Employed by HuBoard (2016 - present)
+
+    - Yoram Harmelin <yoram.harmelin@gmail.com>, active 2012 - 2014
+      Employed by AXA (2012 - 2014)
+
+    - Jameson Miller <jamill@microsoft.com>, active 2012 - 2015
+      Employed by Microsoft Corp (2012 - 2015)
+
+    - Marius Ungureanu <marius.ungureanu@xamarin.com>, active 2014 - present
+      Employed by Xamarin Inc (2014 - 2016)
+      Employed by Microsoft Corp (2016 - present)
+
+    Contributors to the NodeGit project (who are not included above):
+
+    - Josh Abernathy <joshaber@gmail.com>, active 2015 - present
+      Employed by GitHub Inc (2015 - present)
+
+    - Tim Branyen <tim@tabdeveloper.com>, active 2011 - present
+      Employed by Netflix (2015 - present)
+      Employed by Bocoup (2011 - 2015)
+
+    - John Haley <johnh@axosoft.com>, active 2014 - present
+      Employed by Axosoft (2014 - present)
```

Comment:
John Haley is no longer with Axosoft and is not an active contributor to NodeGit. This would've happened in 2017.